critiquebrainz/frontend/external/bookbrainz_db/literary_work.py

```diff
@@ -107,7 +107,7 @@ def fetch_multiple_literary_works(bbids: List[uuid.UUID], work_type=None, limit=
             for literary_work in literary_works:
                 literary_work = dict(literary_work)
                 literary_work['identifiers'] = fetch_bb_external_identifiers(literary_work['identifier_set_id'])
-                literary_work['rels'] = fetch_relationships(literary_work['relationship_set_id'], ['Edition'])
```

Comment:
Do we also want to add an additional section to this page, "Edition Groups in which this work appears"? @MonkeyDo

Reply:
I've added this section